README.md

```diff
@@ -1,91 +1,64 @@
 Log Cache Release
 =================
 
-Log Cache Release is a [bosh](https://github.com/cloudfoundry/bosh) release
-for [Log Cache](https://code.cloudfoundry.org/log-cache). It provides
-an [in memory caching layer](https://docs.google.com/document/d/1yhfl0EB_MkHkh4JdRZXGeQMx_BDMCuB-SpPuSrD3VOU/edit#) as a replacement for `cf logs --recent` and container metrics retrieval.
+## What is Log Cache?
 
-### Deploying Log Cache
+Log Cache provides an in memory caching layer for logs and metrics which are emitted by
+applications and system components in Cloud Foundry. 
 
-Log Cache can be deployed within
-[Cloud Foundry](https://github.com/cloudfoundry/cf-deployment).
+Log Cache is deployed as a group of nodes which communicate between themselves
+over GRPC. Source IDs (such as the CF application ID or a unique string for a
+system component) are hashed to determine which Log Cache
+instance will cache the data. Envelopes from a source ID can be sent to any log cache instance and
+are then forwarded to the assigned node. Similarly queries for Log Cache can be
+sent to any node and the query will be forwarded to the appropriate node which
+is caching the data for the requested source ID.
 
-Log Cache relies on Loggregator and reads data from the Reverse Log Proxy.
+## How does Log Cache fit into Cloud Foundry?
 
-#### Cloud Config
+Log Cache is included by default in 
+[Cloud Foundry's cf-deployment](https://github.com/cloudfoundry/cf-deployment).
 
-Every bosh deployment requires a [cloud
-config](https://bosh.io/docs/cloud-config.html). The Log Cache deployment
-manifest assumes the CF-Deployment cloud config has been uploaded.
+By default Log Cache receives data from syslog agents which is an add on which runs on all jobs by default. Log Cache can also be configured
+to read from the Reverse Log Proxy instead, though this option is not recommended because of firehose scalability limits.
 
-#### Creating and Uploading Release
+Log Cache is queried by Cloud Controller for app instance metrics such as CPU usage and memory when retrieving details for applications and 
+by the cf cli directly to retrieve recent logs. It can also be queried using the Log Cache CLI plugin to retrieve system component metrics.
 
-The first step in deploying Log Cache is to create a release. Final releases
-are preferable, however during the development process dev releases are
-useful.
 
-The following commands will create a dev release and upload it to an
-environment named `lite`.
+## How do I configure it?
 
-```
-bosh create-release --force
-bosh -e lite upload-release --rebase
-```
+### Scaling
 
-#### Cloud Foundry
+We recomend aiming for retaining 15 minutes of logs and metrics in log-cache per source. This can be monitored with the `cf log-meta` command
```

Comment:
IDK if this comment is valuable/what we want. For regular cf-cli use I suspect ranges in seconds might be fine for regular use. For usecases of metric querying and debugging, it's a lot bigger and harder of a question.

Reply:
Which is largely to say, to a large degree, the retention period only matters because someone who cares wants a higher retention period, it's not something we maybe need to give recomendations on.